src/featureFlags/flags/dummyFeatureGate.ts

```diff
@@ -0,0 +1,9 @@
+import { BaseVariant, FeatureFlag, useBaseFlag } from '../index'
+
+function useDummyGateFlag(): BaseVariant {
+  return useBaseFlag(FeatureFlag.statsigDummy, BaseVariant.Control)
```

Comment:
nit: you do not need to pass BaseVariant.Control as a default, as this _is_ the default.

```suggestion
  return useBaseFlag(FeatureFlag.statsigDummy)
```